### ISSUE TITLE
Add possibility to use cards from lockscreen without passcode

### ISSUE THREAD

**Issue by Mannshoch** [brarcher/loyalty-card-locker#242]
At moment this App is not usefull for me because i have to unlock my phone everytime i wish to use a card.
Please made it (or specific cards) usable without unlocking my phone.

**Comment by brarcher** [comment]
Can you give an example of how you might expect it to work? Do you mean [lock screen widgets](https://fieldguide.gizmodo.com/put-your-key-apps-on-the-android-lock-screen-for-easy-a-1621573474)?

**Comment by Mannshoch** [comment]
Yes I mean the lock screen widget. At moment, I also have Loyalty-card on my lock screen but If I start it I have to login with my passcode.

**Comment by devurandom** [comment]
In https://github.com/brarcher/loyalty-card-locker/issues/287#issuecomment-452351465 @sphh suggested that [PassAndroid](https://github.com/ligi/PassAndroid) and [Transportr](https://github.com/grote/Transportr) already have similar features. Maybe code can be borrowed / shared?
